### PR TITLE
Fix bad hyperlink in `cupy.around` documentation

### PR DESCRIPTION
**Pull request by asi1024**
Related to issue #1834 (PR: #1835).

https://docs-cupy.chainer.org/en/latest/reference/generated/cupy.around.html
https://docs-cupy.chainer.org/en/latest/reference/generated/cupy.round_.html

### CODE REVIEW

**Comment by asi1024** [comment]
jenkins, test this please.

File: cupy/core/fusion.py [hunk 1]
```diff
@@ -586,8 +586,9 @@ def make_fusion_var(var, ndim):
         # Make FusionVar list
         var_list = [self._get_fusion_var(_) for _ in args]
         if 'out' in kwargs:
-            out_var = self._get_fusion_var(kwargs.pop('out'))
-            var_list.append(out_var)
+            out = kwargs.pop('out')
```
Comment (kmaehashi) [inline cupy/core/fusion.py]: Is this change related to this PR?

**Comment by okuta** [comment]
LGTM!
